.github/workflows/build-wheel-linux-x86_64.yaml

```
@@ -259,7 +258,7 @@ jobs:
               -DPYTHON_EXECUTABLE=$(which python${{ matrix.python_version }}) \
               -Dpybind11_DIR=$(python${{ matrix.python_version }} -c "import pybind11; print(pybind11.get_cmake_dir())") \
               -DENABLE_LIGHTNING_KOKKOS=ON \
-              -DLIGHTNING_GIT_TAG="master" \
+              -DLIGHTNING_GIT_TAG="latest_release" \
```

Do you want to also pin this to a particular git flag so that when we get back to this commit, we would be able to build catalyst with lightning?
 
```suggestion
              -DLIGHTNING_GIT_TAG="v0.33.0" \
```

---

This is for the wheel action, so I don't think the pinning is relevant here. It's mainly needed for our dev build recipe.